customers.sql

```diff
@@ -0,0 +1,33 @@
+with customer_orders as (
+ select
+   customers.id as customer_id,
+   customers.name,
+   customers.email,
+   -- add coalesce if you do a left join to get customers with 0 orders
+   count(orders.id) as number_of_orders
+ from `analytics-engineers-club.coffee_shop.customers` customers
+ -- could left join to get customers who don't have any orders 
+ inner join `analytics-engineers-club.coffee_shop.orders` orders
```

Comment:
Helpful comment here! Assuming in this case you want the table to _only_ contain customers orders 👍